package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "blueprint-markdown-renderer",
   "nicename": "Blueprint Markdown Renderer",
-  "version": "1.2.0",
+  "version": "0.0.0-development",
```

Comment:
And this line is the only problem I see with `semantic-release` usage.

You don't see version in the `package.json` file, and you won't see the real published/uploaded (to npm) file `package.json` until you download/install such version.

Is it possible to somehow overcome this burden = keep bumping versions in git, or not?
Also - it looks pretty good when you have commit history with the `bump version` -- it's easy to search for those. But YMMV.